src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
```

Comment:
Ah. That sounds reasonable. Also looks like we do have some tests for `label` and `hintLabel` (mostly just `label` though)